src/app/components/tableTranslation/TableTranslation.tsx

```diff
@@ -0,0 +1,92 @@
+"use client";
+import { createTranslatProject } from '@/app/service/DictionariesServices';
+import { getAllProjects } from '@/app/service/ProjectServices';
+import { schemaCreateTranslation } from '@/app/utils/TranslationValidation';
+import { yupResolver } from '@hookform/resolvers/yup';
+import { Box, Button, Table, TableBody, TableCell, TableContainer, TableHead, TableRow, TextField, Typography } from '@mui/material';
+import { useForm } from 'react-hook-form';
+import { useMutation, useQuery, useQueryClient } from 'react-query';
+import { toast } from 'react-toastify';
+import styles from "./style.module.scss";
+
+interface TranslatData {
+  key: string;
+  text: string;
+}
+
+const TableTranslation = ({ projectsId }: { projectsId: any }) => {
```

Comment:
you use projectsId here as just id which is string and use it in tabs component and was object , don't do that ever, you should make name unique just for one meaning.